test/mechanics/integrands/NeumannBc.cpp

```diff
@@ -80,324 +80,61 @@ BOOST_AUTO_TEST_CASE(NeumannBc1Din2D)
     BoostUnitTest::CheckEigenMatrix(hessian0(dof, dof), Eigen::MatrixXd::Zero(4, 4));
 }
 
-BOOST_AUTO_TEST_CASE(NeumannBc2Din3D_TriangleLinear)
+
+BOOST_AUTO_TEST_CASE(NeumannBc2Din3D)
 {
     std::vector<NodeSimple> nodesDisp;
-    for (int i = 0; i < 4; i++)
+    for (int i = 0; i < 3; i++)
         nodesDisp.push_back(NodeSimple(Eigen::Vector3d(0, 0, 0)));
 
     std::vector<NodeSimple> nodes;
     nodes.push_back(NodeSimple(Eigen::Vector3d(1, 0, 0)));
     nodes.push_back(NodeSimple(Eigen::Vector3d(0, 1, 0)));
     nodes.push_back(NodeSimple(Eigen::Vector3d(0, 0, 1)));
```

Comment:
The element has a `std::initializerlist` ctor. So you could do
~~~cpp
NodeSimple n0(Eigen::Vector3d(1, 0, 0));
NodeSimple n1(Eigen::Vector3d(0, 1, 0));
NodeSimple n2(Eigen::Vector3d(0, 0, 1));
// ...
ElementCollectionFem element({{n0, n1, n2}, coordinateInterpolation});
~~~
which is a bit shorter and avoids the `notePtrs` vector